### PR TITLE
Update commands in user guide

### PR DESCRIPTION
**Pull request by richavm14**

### CODE REVIEW

File: docs/UserGuide.md [hunk 1]
```diff
@@ -205,22 +205,21 @@ Coming Soon!
...
-**Team Delink** | `team delink t/TEAM-NAME m/TEAM-MEMBER...` <br> e.g. `team delink t/UIDevelopers m/John m/Jane`
-**Team Link** | `team link t/TEAM-NAME m/TEAM-MEMBER...` <br> e.g. `team link t/UIDevelopers m/John m/Jane`
-**Team Rename** | `team rename o/TEAM-NAME n/NEW-TEAM-NAME` <br> e.g. `team rename o/UIDevelopers n/UnitTesters`
-**View** | `view t/TEAM-NAME` <br> e.g.,`view t/UIDevelopers`
-**View All** | `view all`
+| Action              | Format, Examples                                                                                                                                            |
+|---------------------|-------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| **Member Add**      | `add n/NAME p/PHONE e/EMAIL a/ADDRESS t/TAG` <br> e.g. `add n/John Doe p/99853657 e/john@gmail.com a/414, North Bridge Ave 5, #09-86 t/friends t/owesMoney` |
+| **Member Delete**   | `delete p/GLOBAL-PERSON-INDEX` <br> e.g. `delete n/1`                                                                                                       |
```
Comment (TJun-Jie) [inline docs/UserGuide.md:211]: Example wrong , should be p instead of n

File: docs/UserGuide.md [hunk 2]
```diff
@@ -205,22 +205,21 @@ Coming Soon!
...
+| **Member unAssign** | `unassign m/MEMBER-INDEX t/TEAM-INDEX` <br> e.g.`unassign m/1 t/1`                                                                                          |
+| **Member List**     | `userlist`                                                                                                                                                  |
+| **Help**            | `help`                                                                                                                                                      |
+| **Task Add**        | `taskadd t/TEAM-INDEX n/TASK-NAME` <br> e.g. `taskadd t/1 n/Finish project`                                                                                 |
+| **Task Deadline**   | `task deadline t/TASK-DESCRIPTION d/DEADLINE` <br> e.g. `task deadline t/Implement a payment system d/12/12/2023`                                           |
+| **Task Delete**     | `taskdelete t/TEAM-INDEX task/TASK-INDEX` <br> e.g. `taskdelete t/Create GUI for the Duke app`                                                              |
```
Comment (TJun-Jie) [inline docs/UserGuide.md:219]: Example wrong

File: docs/UserGuide.md [hunk 3]
```diff
@@ -205,22 +205,21 @@ Coming Soon!
...
+| **Task Mark**       | `taskmark t/TEAM-INDEX task/TASK-INDEX` <br> e.g. `taskmark t/1 t/1`                                                                                        |
```
Comment (TJun-Jie) [inline docs/UserGuide.md:220]: Example format wrong

File: docs/UserGuide.md [hunk 4]
```diff
@@ -205,22 +205,21 @@ Coming Soon!
...
+| **Task Mark**       | `taskmark t/TEAM-INDEX task/TASK-INDEX` <br> e.g. `taskmark t/1 t/1`                                                                                        |
```
Comment (TJun-Jie) [inline docs/UserGuide.md:221]: Example format wrong

File: docs/UserGuide.md [hunk 5]
```diff
@@ -205,22 +205,21 @@ Coming Soon!
...
+| **Task unMark**     | `taskunmark t/TEAM-INDEX task/TASK-INDEX` <br> e.g. `taskunmark t/1 t/1`                                                                                    |
```
Comment (TJun-Jie) [inline docs/UserGuide.md:222]: Example format wrong

**Review by TJun-Jie — CHANGES_REQUESTED** [review]

File: docs/UserGuide.md (hunk 1 above)
Comment (richavm14) [reply docs/UserGuide.md]: Have updated in recent commit!

File: docs/UserGuide.md (hunk 2 above)
Comment (richavm14) [reply docs/UserGuide.md]: Have updated in recent commit!

File: docs/UserGuide.md (hunk 3 above)
Comment (richavm14) [reply docs/UserGuide.md]: Have updated in recent commit!

File: docs/UserGuide.md (hunk 4 above)
Comment (richavm14) [reply docs/UserGuide.md]: Have updated in recent commit!

File: docs/UserGuide.md (hunk 5 above)
Comment (richavm14) [reply docs/UserGuide.md]: Have updated in recent commit!

**Pull request merged**